conftest.py

```diff
@@ -1,6 +1,19 @@
 # https://pytest.org/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option
+import os
 import pytest
 
+
+# Enable more logging and checks (https://docs.python.org/3/library/asyncio-dev.html)
+os.environ["PYTHONASYNCIODEBUG"] = "1"
```

Comment:
You mentioned that this makes things run a bit slowly.  Is there a cost to this?

Reply:
With Tornado 4.5.2 (PollIOLoop) I get: 948 passed, 91 skipped, 9 xfailed, 3 xpassed, 1 warnings in 452.45 seconds
With Tornado git master (AsyncIOLoop, debug enabled) I get: 2 failed, 939 passed, 98 skipped, 11 xfailed, 1 xpassed, 1 warnings in 749.41 seconds
With Tornado git master (AsyncIOLoop, debug disabled) I get: 1 failed, 940 passed, 98 skipped, 8 xfailed, 4 xpassed, 1 warnings in 394.44 seconds

(this is Python 3.6, without the slow tests)

Moreover, it seems that in debug mode, the py.test process' RSS balloons up to 3GB at the end of the test suite. So we'd better disable debug mode after all.